src/OperationKeyboard/controller.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+import { EventDispatcher } from '../eventDispatcher.js';
+import viewTemplate from './view.hbs';
+import stringToDom from 'string-to-dom';
+
+export const OperationKeyboardEvents = {
+    RESET_ON_NEXT_INPUT: 'resetOnNextInput',
+    GET_OPERATION: 'getOperation'
+};
+
+export class OperationKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindOpsEvents();
+    }
+    /**
+     * Binds event listeners to operation buttons.
+     */
+    _bindOpsEvents() {
+        let viewString = viewTemplate();
+        this.view = stringToDom(viewString);
+        let operationButtons = this.view.querySelectorAll('.operation');
+        for (let i = 0; i < operationButtons.length; i++) {
+            operationButtons[i].addEventListener("click", (event) => {
+                this.trigger(OperationKeyboardEvents.RESET_ON_NEXT_INPUT)
```

Comment:
By throwing of this event, you are basically assume that ops controller is aware of screen and that it should reset on next input. This is not an optimal architecture since your controllers are connected -- the state which we are trying to avoid.

You controller should just throw an event 'hey a user has clicked on that operation'. What his parent is going to do with that information, is not a concern of this controller.

Additional complication is that you are throwing two events here, and one is enough. Or in another words, once a user clicks '+' and calculator is informed about that, it can reset input and do other things.